source/includes/_claim_fund_api.md

```diff
@@ -0,0 +1,1126 @@
+  "recipient_name": "Recipient",
+  "note": "note",
+  "expiration": "10"
+});
+
+var xhr = new XMLHttpRequest();
+xhr.withCredentials = true;
+
+xhr.addEventListener("readystatechange", function() {
+  if(this.readyState === 4) {
+    console.log(this.responseText);
+  }
+});
+
+xhr.open("POST", "%7B%7Bbase_url%7D%7D/api/claim-fund");
+xhr.setRequestHeader("Content-Type", "application/json");
+xhr.setRequestHeader("Accept", "application/json");
+xhr.setRequestHeader("x-oy-username", "{{username}}");
+xhr.setRequestHeader("x-api-key", "{{api-key}}");
+
+xhr.send(data);
+```
+
+```php
+<?php
+require_once 'HTTP/Request2.php';
+$request = new HTTP_Request2();
+$request->setUrl('{{base_url}}/api/claim-fund');
+$request->setMethod(HTTP_Request2::METHOD_POST);
+$request->setConfig(array(
+  'follow_redirects' => TRUE
+));
+$request->setHeader(array(
+  'Content-Type' => 'application/json',
+  'Accept' => 'application/json',
+  'x-oy-username' => '{{username}}',
+  'x-api-key' => '{{api-key}}'
+));
+$request->setBody('{
+  \"amount\": 500000,
+  \"partner_trx_id\": \"TRX-20250408-001\",
+  \"recipient_email\": \"recipient@mail.com\",
+  \"recipient_name\": \"Recipient\",
+  \"note\": \"note\",
+  \"expiration\": \"10\"
+}');
+try {
+  $response = $request->send();
+  if ($response->getStatus() == 200) {
+    echo $response->getBody();
+  }
+  else {
+    echo 'Unexpected HTTP status: ' . $response->getStatus() . ' ' .
+    $response->getReasonPhrase();
+  }
+}
+catch(HTTP_Request2_Exception $e) {
+  echo 'Error: ' . $e->getMessage();
+}
+```
+
+```python
+import http.client
+import json
+
+conn = http.client.HTTPSConnection("{{base_url}}")
+payload = json.dumps({
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "expiration": "10"
+})
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'x-oy-username': '{{username}}',
+  'x-api-key': '{{api-key}}'
+}
+conn.request("POST", "/api/claim-fund", payload, headers)
+res = conn.getresponse()
+data = res.read()
+print(data.decode("utf-8"))
+```
+
+> The above command returns JSON structured similar like this:
+
+```json
+{
+  "status": {
+    "code": "000",
+    "message": "Success"
+  },
+  "tx_status": "INITIATED",
+  "tx_status_description": "The Claim Fund link has successfully been created and is awaiting the end user's response",
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "claim_fund_link": "https://claimfund.oyindonesia.com/claim-fund/claim/251280aa-95cb-41f2-84c3-8d2b247aa2b4?token=1ac006ce-82d9-4cb6-9afd-a621ca8dafa0",
+  "expiration": "2025-04-08 12:10:00",
+  "created": "2025-04-08 02:10:00",
+  "updated": "2025-04-08 02:10:00"
+}
+
+```
+
+Use this API to to create a new Claim Fund transaction.
```

Comment:
format copywritingnya disamain aja dengan yg lain:
`This endpoint is used to create a new Claim Fund API transaction.` @guntoroyk